### DISCUSSION TITLE
Second italic word in title gets upper case in BibLatex export

### DISCUSSION THREAD

**Discussion by eberlejonas** [retorquere/zotero-better-bibtex#3014] [General]
Hi,

in my Zotero library, I have entries containing species names which are composed of an upper case genus epthet and a lower case species epithet. Both should be in italics.

In the Zotero title field, it might look like this:
Kairomone response in <i>Thanasimus</i> predators to pheromone components of <i>Ips typographus</i>

In my file that I export via Better BibLaTeX, I get the following:
title = {Kairomone Response in {{{\emph{Thanasimus}}}} Predators to Pheromone Components of {{{\emph{Ips}}}}{\emph{ Typographus}}}
Note the upper case last word.

I expect it to be lower case and both word to be in one \emph-command: {\emph{Ips typographus}}

Am I missing any setting? Thank you!

**Comment by retorquere** [comment]
I'll need a debug log for that item. Right-click, BBT, send debug log.
